TECHMANIA/Assets/Scripts/UniversalIO.cs

```diff
@@ -1,27 +1,92 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using UnityEngine;
 
 // Generalized IO that works on both a file system and
 // streaming assets.
 public static class UniversalIO
 {
-    public static bool FileExists(string path)
+    public const string ANDROID_CONTENT_URI = "content://";
+
+    public static string GetRealPathFromUri(string path)
```

Comment:
While it is important, it's also difficult for a non-Android developer to understand what this method does from its name. How about `GetAndroidRealPath`?